Update direct-routing-plan.md (#3008)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/direct-routing-plan.md

- Microsoft Phone System 
- Microsoft Teams + Skype for Business Plan 2 if included in Licensing Sku

> - Microsoft Audio Conferencing (please read the notes and the paragraph below to see specific examples whed the license is requried)

Reviewer: ```suggestion
- Microsoft Audio Conferencing (please read the notes and the paragraph below for specific examples and information about the license requirement)
```

> [!IMPORTANT]
>  In the case that you would like to add external participants to scheduled meetings, either by dialing out to them or by providing the dial-in number, the audio conferencing license is *required*.

> Ad Hoc call escalation and Audio Conferencing license.

Reviewer: ```suggestion
Ad hoc call escalation and Audio Conferencing license.
```

> A Teams User can start a one to one Teams to PSTN or Teams to Teams call and add a PSTN participant to it. This scenario called ad hoc conference. The path the call takes, depends if the Microsoft Audio Conferencing license assigned to the user who escalates the call or not.

Reviewer: ```suggestion
A Teams user can start a one on one Teams to PSTN or Teams to Teams call and add a PSTN participant to it. This scenario is called an ad hoc conference. The path that the call takes depends whether the Microsoft Audio Conferencing license assigned to the user escalates the call or not.
```

> 1. If the Teams user, who escalates the call has the Microsoft Audio Conferencing license assigned, the escalation happens via Microsoft Audio Conferencing service. The remote PSTN participant who is being invited to the existing call on notification about incoming call sees the number of the Microsoft bridge, assigned to the Teams user who initiated the escalation

Reviewer: ```suggestion
1. If the Teams user who escalates the call has a Microsoft Audio Conferencing license assigned, the escalation happens via the Microsoft Audio Conferencing service. The remote PSTN participant who is invited to the existing call receives a notification about the incoming call and sees the number of the Microsoft bridge assigned to the Teams user who initiated the escalation.
```

> 2. If the Teams user, who escalates the call does not have the Microsoft Audio Conferencing license assigned, the escalation happens via a Session Border Controller, connected to Direct Routing interface. The remote PSTN participant who is being invited to the call on notification about incoming call sees the number of the Teams User who initiated the escalation. The specific SBC, used for the escalation, defined by Routing Policy of the user. 

Reviewer: ```suggestion
2. If the Teams user who escalates the call does not have the Microsoft Audio Conferencing license assigned, the escalation happens via a Session Border Controller connected to the Direct Routing interface. The remote PSTN participant who is invited to the call receives a notification about the incoming call and sees the number of the Teams user who initiated the escalation. The specific SBC, used for the escalation, is defined by Routing Policy of the user. 
```
